ui/app/[locale]/(authorized)/kafka/[kafkaId]/consumer-groups/[groupId]/reset-offset/ResetOffset.tsx

```diff
@@ -219,11 +238,15 @@ export function ResetOffset({
               <Button
                 variant="primary"
                 onClick={handleSave}
-                isDisabled={isLoading}
+                isDisabled={isLoading || !isDisabled}
               >
                 {t("save")}
               </Button>
-              <DryrunSelect openDryrun={openDryrun} cliCommand={cliCommand} />
+              <DryrunSelect
+                openDryrun={openDryrun}
+                cliCommand={cliCommand}
+                isDisabled={!isDisabled}
```

Comment:
This seems somewhat confusing. The `DryrunSelect` is disabled when the criteria here are "not disabled". Should the variable in this file be named differently, or am I missing something?